libndt.hpp

```diff
@@ -1953,15 +2010,47 @@ bool Client::ndt7_upload() noexcept {
   for (;;) {
     auto now = std::chrono::steady_clock::now();
     std::chrono::duration<double> elapsed = now - begin;
+    std::chrono::duration<double, std::micro> elapsed_ms =
+      std::chrono::duration_cast<std::chrono::microseconds>(elapsed);
     if (elapsed.count() > max_upload_time) {
       LIBNDT_EMIT_DEBUG("ndt7: upload has run for enough time");
       break;
     }
     constexpr auto measurement_interval = 0.25;
     std::chrono::duration<double> interval = now - latest;
     if (interval.count() > measurement_interval) {
+      nlohmann::json measurement;
+      measurement["AppInfo"] = nlohmann::json();
+      measurement["AppInfo"]["ElapsedTime"] = elapsed_ms.count();
```

Comment:
```JSON
{"AppInfo":{"ElapsedTime":9823361.0}}
```

Do you know whether this will parse? In `ndt-server` we define `ElapsedTime` as:

```Go
type AppInfo struct {
  ElapsedTime int64
}
```

Reply:
So, I checked and it does not parse :-(.